recipes/mar_greenland/meta.yaml

```diff
@@ -0,0 +1,19 @@
+title: 'MAR Greenland'
+description: 'Analysis-ready Zarr datasets derived from MAR netcdfs'
+recipes:
+  - id: mar-greenland
+    object: 'recipe:recipe'
+provenance:
+  providers:
+    - name: 'Google Bucket ldeo-glaciology'
+      description: 'Data repository for the Polar Geophysics Group at Lamont-Doherty Earth Observatory, Columbia University'
+      roles:
+        - licensor
+      url: https://storage.googleapis.com/ldeo-glaciology/MAR/6.5km/
```

Comment:
Is there another website where the dataset is described by any chance?

Reply:
The original data is on a private, password-protected FTP. We've moved the data to this google bucket to avoid authorization issues in the recipe. The model and data are best described in this paper though: https://tc.copernicus.org/articles/11/1015/2017/

Reply:
👍🏽 for including this reference in the metadata

Reply:
Do you mean it's better to change the google bucket url to the paper url?

Reply:
> Do you mean it's better to change the google bucket url to the paper url?

yes